.circleci/config.yml

```diff
@@ -48,6 +43,13 @@ jobs:
       # resetting the working tree avoids lerna errors due to unclean working trees
       - run: git reset --hard
       - run: yarn run lerna:publish
+      - run: cd packages/xml-for-vscode
+      - run: yarn run vsix:*

```

Comment:
i think you need to do the `cd ...` in the same line, it is not exactly like shell
- https://discuss.circleci.com/t/can-i-run-npm-commands-in-subfolders/21257/3

Reply:
- does yarn recognize `vsix:*` command?
  I thought it was an `npm-run-all` feature.